### ISSUE TITLE
Accessibility violations on sign-in page (duplicate element IDs)

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#7509]
### System details

    RStudio Edition : Server
    RStudio Version : 1.3.1073, 1.4.660
    OS Version      : Any
    R Version       : Any

### Steps to reproduce the problem
Load the sign-in page, and run an accessibility checker such as Axe or even Chrome Lighthouse, OR save the page HTML and run HTML validation on it.

### Describe the problem in detail
Invalid HTML due to duplicate element ID "caption" used in two places, same with ID "buttonpanel".

### Describe the behavior you expected
Valid HTML; having duplicate element IDs causes problems for screen readers and is a severe accessibility violation (http://www.w3.org/TR/WCAG20/#ensure-compat-parses). This page was accessibility-clean in initial 1.3 release (1.3.959).

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by gtritchie** [comment]
Probably doesn't meet the bar for 1.3, even though it is a regression from initial 1.3 release. The sign-in page is very simple, and I have not been able to identify any actual screen-reader problems caused by this error. Should definitely fix for 1.4.

**Comment by rich-rstudio** [comment]
verified on RSP and open source server 1.4.1049 using Axe
